src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -1025,12 +1031,10 @@ public static function xpathLiteral($s)
      */
     private function filterRelativeXPath(string $xpath)
     {
-        $prefixes = $this->findNamespacePrefixes($xpath);
-
+        $domxpath = $this->createDOMXPath($this->document, $this->findNamespacePrefixes($xpath));
```

Comment:
are we sure all nodes share the same document? 

note: Travis test is failing here

Reply:
There is not necessarily a document on that point. In the tests (mainly) because of an empty content. This isn't hard to handle. I'll check tests and all the possible paths tomorrow.